.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
     - $HOME/.pip-cache/
 
 python:
-  - "3.4"
+  - "3.6"
 
 virtualenv:
   system_site_packages: true
```

Comment:
The CI is failing because travis is not supporting this system_site_packages for 3.6

Reply:
You're right, thanks!. Done.

Now it's green.